packages/teleterm/src/ui/services/quickInput/quickInputService.ts

```diff
@@ -45,28 +49,34 @@ export class QuickInputService extends Store<State> {
       launcher,
       serviceClusters
     );
-    this.quickCommandPicker = new pickers.QuickCommandPicker();
+    this.quickCommandPicker = new pickers.QuickCommandPicker(launcher);
     this.setState({
-      picker: this.quickCommandPicker,
       inputValue: '',
     });
+
+    const sshLoginPicker = new pickers.QuickSshLoginPicker(
+      serviceWorkspaces,
+      serviceClusters
+    );
+
+    this.quickCommandPicker.registerPickerForCommand(
+      'tsh ssh',
+      new pickers.QuickTshSshPicker(launcher, sshLoginPicker)
+    );
+    this.quickCommandPicker.registerPickerForCommand(
+      'tsh proxy db',
+      new pickers.QuickTshProxyDbPicker(launcher)
+    );
   }
 
   state: State = {
-    picker: null,
     inputValue: '',
     visible: false,
   };
 
+  // TODO: There's no "back" in the new command bar. We can probably just remove this method and the
```

Comment:
I'll get rid of all the `TODO`s that don't have my username next to it before I merged this PR.